src/helpers.ts

```diff
@@ -473,26 +475,37 @@ export function getShuffleItems(
     userId: string,
     item: BaseItemDto
 ): Promise<BaseItemDtoQueryResult> {
-    const query: ItemQuery = {
-        Fields: requiredItemFields,
-        Filters: 'IsNotFolder',
-        Limit: 50,
-        Recursive: true,
-        SortBy: 'Random',
-        UserId: userId
+    const query: ItemsApiGetItemsRequest = {
+        fields: requiredItemFields,
+        filters: ['IsNotFolder'],
+        limit: 50,
+        recursive: true,
+        sortBy: ['Random'],
+        userId
     };
 
+    let additionalParams: ItemsApiGetItemsRequest;
```

Comment:
I don't see a need for additionalParams, you can just modify the query

Reply:
All properties have the `readonly` keyword applied in `ItemsApiGetItemsRequest` so they cannot be modified after the query is created. But maybe there's a pattern to use here that I'm not aware of...

Reply:
I wasn't aware that the properties are all readonly. I would then write it like so:

```
let query = {...};
// ...
query = {...query, additionalProp: true};
```

So replace the query variable with the additional properties added